### PR TITLE
Hulya - Welcome page

### LINKED ISSUE #63: Create Welcome Page

### PR DESCRIPTION
**Pull request by hulyak**
## Description

- designed the welcome page according to Figma 
- created custom styles for Typography
- used flexbox layout

Note: I think we can have more space between the lettuce image and GreenButton. I have tried to edit GreenButton, but styling didn't apply. Maybe we can play with it, later!

### Edits:

- removed GreenButton.css, styling is not working.
- edited button styling inside `src/themes/theme.js`
- created `20px` margin for green button

## Related Issue

closes #63 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

Screen Shot 2021-09-01 at 7 09 10 PM

### After

Screen Shot 2021-09-01 at 7 05 07 PM
Screen Shot 2021-09-01 at 7 05 26 PM
Screen Shot 2021-09-01 at 7 05 39 PM

## Testing Steps / QA Criteria

- pull down `welcome-page` branch
- `npm start`
- see the changes in welcome page

### CODE REVIEW

**Comment by isaabutaa** [comment]
Nice job with this @hulyak! I see you added some more typography styling to keep it consistent throughout - that's awesome. In regards to increasing the margin for the GreenButton, you could do it by editing the styling for `MuiButton` inside of src/themes/theme.js:

`MuiButton: {`
      `root: {`
        `borderRadius: '25px',`
        `margin: '8px',  // you could edit the margin here`
      `},`
    `},` 

I probably should have gotten rid of the GreenButton.css file because it doesn't actually work. 🤦‍♂️

File: src/styles/GreenButton.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .green-btn {
-  margin: 16px;
+  margin: 5rem;
 }
```
Comment (isaabutaa) [inline src/styles/GreenButton.css]: If you do end up changing the margin for the button (which I'm totally okay with), I might suggest making it smaller than `5rem`, or changing it to `2rem` or `16px` or something. You should be able to make changes to the margin of the button in `themes.js`.

**Review by isaabutaa — APPROVED** [review]

**Review by sandaiiyahh — APPROVED** [review]
Thank you for doing this Hulya! It looks great! :))

File: src/styles/GreenButton.css (hunk 1 above)
Comment (hulyak) [reply src/styles/GreenButton.css]: I was trying to check if it is adding margin, but it didn't apply I think we can delete it. I will add margin inside the `themes.js` file.

**Review by ksiman14 — APPROVED** [review]
Thanks for doing this @hulyak, it looks so cute!

**Pull request merged**